src/components/Modals/Send/Form.tsx

```diff
@@ -62,6 +62,7 @@ export const Form: React.FC<SendFormProps> = ({ asset: initialAsset, accountId }
 
   const handleAssetSelect = async (asset: Asset) => {
     methods.setValue(SendFormFields.Asset, { ...asset, ...marketData })
+    methods.setValue(SendFormFields.Input, '')
```

Comment:
reset the user facing `Input` field on asset change, which in tun resets the internal `Address` and `VanityAddress` fields used later